raiwidgets/requirements-dev.txt

```diff
@@ -9,9 +9,6 @@ requirements-parser==0.2.0
 
 wheel
 
-# Required for interpret-community 0.18.0
-lightgbm==2.3.0
```

Comment:
note: this is already specified in requirements.txt for raiwidgets and isn't needed here